### PR TITLE
Feature/extension version info

### PR DESCRIPTION
**Pull request by igorkol**

### CODE REVIEW

**Comment by igorkol** [comment]
[Add chrome extension version to the header](https://trello.com/c/0WAYbhPO/10-add-chrome-extension-version-to-the-header)

File: background.js [hunk 1]
```diff
@@ -60,35 +263,41 @@ MESSAGING FROM CONTENT SCRIPTS
...
   // First, validate the message's structure.
   if ((msg.from === 'content') && (msg.subject === 'showPageAction')) {
     // Enable the page-action for the requesting tab.
     chrome.pageAction.show(sender.tab.id);
   }
   
+  //Clicking the Extension's Page Action
+  if ((msg.from === 'popup') && (msg.subject === 'pageActionClicked')) {
+    //Just in case we need it later...
+    console.log('Page action clicked.');
```
Comment (edwardupton) [inline background.js:275]: Let's not console.log anything in production, as it pollutes the console for other kinds of debugging. You can use console.debug if you really need

File: background.js [hunk 2]
```diff
@@ -60,35 +263,41 @@ MESSAGING FROM CONTENT SCRIPTS
...
+    chrome.storage.local.get('errorLog', function (result) {
+      console.log('Sending log to popup.');
```
Comment (edwardupton) [inline background.js:277]: ditto

File: background.js [hunk 3]
```diff
@@ -60,35 +263,41 @@ MESSAGING FROM CONTENT SCRIPTS
...
+      chrome.runtime.sendMessage({ from: 'background', subject: 'updateContent', data: result.errorLog });
+    });
+  }
+
+ //Clicking the Extension's Reset Button
+ if ((msg.from === 'popup') && (msg.subject === 'pageResetClicked')) {
+  console.log('Page reset clicked.');
```
Comment (edwardupton) [inline background.js:284]: ditto

File: background.js [hunk 4]
```diff
@@ -60,35 +263,41 @@ MESSAGING FROM CONTENT SCRIPTS
...
+  resetLocalStorageContent();
+}
+
+  //Turning the extension on and off
   if ((msg.from === 'content') && (msg.subject === 'changeExtensionIcon')) {
-    console.log("Tab ID: " + sender.tab.id + " | Mode: " + msg.mode);
+    console.log("Mode: " + msg.mode);
```
Comment (edwardupton) [inline background.js:290]: ditto

File: background.js (hunk 1 above)
Comment (igorkol) [reply background.js:275]: background.js and it's output and logs is not seen by the end user. it is not outputting that log to the frontend nor user's console, but rather to a hidden dev-only window that monitors the background ops

File: background.js (hunk 2 above)
Comment (igorkol) [reply background.js:277]: same here

File: background.js (hunk 3 above)
Comment (igorkol) [reply background.js:284]: same here

File: background.js (hunk 4 above)
Comment (igorkol) [reply background.js:290]: same for this

**Review by edwardupton — APPROVED** [review]

**Pull request merged**